### DISCUSSION TITLE
Code is not searchable in forks

### DISCUSSION THREAD

**Discussion by peacechen** [material-table-core/core#23] [General]
Github doesn't support searching code in forks. For example:
https://github.com/material-table-core/core/search?q=datagrid&type=code

... returns "Sorry, forked repositories are not currently searchable.".  This won't be sustainable long term as the code diverges more and more from the parent repo. Are there any work-arounds short of creating an entirely new repo from scratch?

**Comment by matthewoestreich** [comment]
What about https://github.com/material-table-core/core/search?q=datagrid

Edit: nvm I take that back
